OvmfPkg/Library/SmmCpuPlatformHookLibQemu/SmmCpuPlatformHookLibQemu.c

```diff
@@ -131,3 +131,24 @@ SmmCpuPlatformHookBeforeMmiHandler (
 {
   return EFI_UNSUPPORTED;
 }
+
+/**
+  Checks if platform specifc LMCE status check is needed.
+
+  This function can be used to check platform specific LMCE status check is needed.
+  By default, it will return TRUE to allow LMCE status check. However,
+  if a platform would like to skip LMCE status check due to any reason, platform can
+  skip LMCE status check by return FALSE.
+
+  @retval TRUE     Platform LMCE status check is needed.
+  @retval FALSE    Platform LMCE status check is skipped.
+
+**/
+BOOLEAN
+EFIAPI
+IsPlatformLmceStatusCheck (

```

Comment:
/**
  Determines if the LMCE status check is needed.

  This function checks whether the LMCE status check is required.
  By default, it returns TRUE, indicating that the LMCE status check
  should be performed. However, if the check should be skipped for
  any reason, it can return FALSE.

  @retval TRUE     LMCE status check is needed.
  @retval FALSE    LMCE status check is skipped.
**/
BOOLEAN
EFIAPI
IsLmceStatusCheckNeeded (
  VOID
  )